pkg/controller/events/event_forwarder.go

```diff
@@ -0,0 +1,93 @@
+package events
+
+import (
+	"context"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type EventHandler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+func (e *EventHandler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	log := ctrl.LoggerFrom(ctx)
+
+	event := &corev1.Event{}
+	if err := e.Get(ctx, req.NamespacedName, event); err != nil {
+		return ctrl.Result{}, err
+	}
+
+	obj := &unstructured.Unstructured{}
+	obj.SetGroupVersionKind(schema.FromAPIVersionAndKind(event.InvolvedObject.APIVersion, event.InvolvedObject.Kind))
+	if err := e.Get(ctx, types.NamespacedName{Name: event.InvolvedObject.Name, Namespace: event.InvolvedObject.Namespace}, obj); err != nil {
+		return ctrl.Result{}, err
+	}
+	ef := obj.GetAnnotations()["vshn.net/forward-events-to"]
+	if ef != "" {
+		log.Info("Forwarding event", "event", event.Name, "to", ef)
+		knc := strings.Split(ef, "/")
+		a := knc[0] + "/" + knc[1]
+		k := knc[2]
+		ns := knc[3]
+		c := knc[4]
+
+		claim := &unstructured.Unstructured{}
+		claim.SetGroupVersionKind(schema.FromAPIVersionAndKind(a, k))
+		if err := e.Get(ctx, types.NamespacedName{Name: c, Namespace: ns}, claim); err != nil {
+			return ctrl.Result{}, err
+		}
+		nev := &corev1.Event{
```

Comment:
We probably want at least `message`, `type` and `reason` as well here? Or otherwise the event is just empty? You can just copy it from the source event.